isites_migration/utils.py

```diff
@@ -275,7 +289,7 @@ def _export_file_repository(file_repository, keyword, topic_title):
                             d_file.write(to_bytes(line, 'utf8'))
             except IOError:
                 logger.exception(
-                    "Failed to export file node %s from file repository % in keyword %s",
+                    u"Failed to export file node %d from file repository %d in keyword %s",
```

Comment:
FWIW, you really don't need to use `%d` in python...`%s` gets you the same results.